specification/maps/data-plane/Route/preview/2024-06-01-preview/route.json

```diff
@@ -0,0 +1,3617 @@
+      "properties": {
+        "isVehicleCommercial": {
+          "description": "Whether the vehicle is used for commercial purposes. Commercial vehicles may not be allowed to drive on some roads.",
+          "type": "boolean",
+          "default": false
+        },
+        "length": {
+          "x-ms-client-name": "lengthInMeters",
+          "description": "Length of the vehicle in meters. A value of 0 means that length restrictions are not considered.",
+          "type": "number",
+          "format": "double",
+          "maximum": 1000000,
+          "minimum": 0,
+          "default": 0
+        },
+        "width": {
+          "x-ms-client-name": "widthInMeters",
+          "description": "Width of the vehicle in meters. A value of 0 means that width restrictions are not considered.",
+          "type": "number",
+          "format": "double",
+          "maximum": 1000000,
+          "minimum": 0,
+          "default": 0
+        },
+        "height": {
+          "x-ms-client-name": "heightInMeters",
+          "description": "Height of the vehicle in meters. A value of 0 means that height restrictions are not considered.",
+          "type": "number",
+          "format": "double",
+          "maximum": 1000000,
+          "minimum": 0,
+          "default": 0
+        },
+        "weight": {
+          "x-ms-client-name": "weightInKilograms",
+          "description": "Weight of the vehicle in kilograms. A value of 0 means that weight restrictions are not considered.",
+          "format": "int64",
+          "type": "integer",
+          "maximum": 1000000,
+          "minimum": 0,
+          "default": 0
+        },
+        "maxSpeed": {
+          "x-ms-client-name": "maxSpeedInKilometersPerHour",
+          "description": "Maximum speed of the vehicle in km/hour. The max speed in the vehicle profile is used to check whether a vehicle is allowed on motorways.\n\nA value of 0 means that an appropriate value for the vehicle will be determined and applied during route planning.\n\nA non-zero value may be overridden during route planning. For example, the current traffic flow is 60 km/hour. If the vehicle maximum speed is set to 50 km/hour, the routing engine will consider 60 km/hour as this is the current situation. If the maximum speed of the vehicle is provided as 80 km/hour but the current traffic flow is 60 km/hour, then routing engine will again use 60 km/hour.",
+          "format": "int64",
+          "type": "integer",
+          "maximum": 250,
+          "minimum": 0,
+          "default": 0
+        },
+        "axleWeight": {
+          "x-ms-client-name": "axleWeightInKilograms",
+          "description": "Weight per axle of the vehicle in kg. A value of 0 means that weight restrictions per axle are not considered.",
+          "format": "int64",
+          "type": "integer",
+          "maximum": 1000000,
+          "minimum": 0,
+          "default": 0
+        },
+        "axleCount": {
```

Comment:
@koyasu221b TomTom Route Matrix v2 does not support axleCount. Can we please cross check all params and enums with TomTom's documentation for sync and async?